lib/go-tc/constants.go

```diff
@@ -98,3 +98,15 @@ func (a AlertLevel) String() string {
 // CachegroupCoordinateNamePrefix is a string that all cache group coordinate
 // names are prefixed with.
 const CachegroupCoordinateNamePrefix = "from_cachegroup_"
+
+// ParameterSecureRead is a string representing the permission to be able to read secure parameters.
+const ParameterSecureRead = "PARAMETER:SECURE-READ"
+
+// SecureServerRead is a string representing the permission to be able to read secure server properties.
+const SecureServerRead = "SECURE-SERVER:READ"
+
+// CDNLocksDeleteOthers is a string representing the permission to be able to delete other users' CDN locks.
+const CDNLocksDeleteOthers = "CDN-LOCK:DELETE-OTHERS"
+
+// ICDNUCDNOverride is a string representing the permission to be able to override the ucdn parameter.
+const ICDNUCDNOverride = "ICDN:UCDN-OVERRIDE"
```

Comment:
IMO it is a little confusing to have other similarly-named things in the `tc` package used for a different purpose

* `ParameterSecureRead`
* `ParametersResponse`
* `CDNLocksDeleteOthers`
* `CDNLocksGetResponse`

and so on.

So either:
* Permissions constants like these should be in their own separate package, or
* The constant names all start with the same prefix like `Perm`

Whether it's `perm.ParameterSecureRead` or `tc.PermParameterSecureRead` (or any other package name or prefix you come up with, it doesn't need to be `perm`), it should be obvious to someone unfamiliar with those specific constants what those constants are used for.

Reply:
fair point, fixed now.